src/index.js

```diff
@@ -234,6 +234,7 @@ OSMBuildings.prototype = {
    * @param {Integer} [options.elevation=<ground height>] - The height above ground to place the model at
    * @param {String} [options.id] - An identifier for the object. This is used for getting info about the object later
    * @param {String} [options.color] - A color to apply to the model
+   * @param {Boolean} [options.noFade] - Display the geojson features immediately (default is to fade-in)
```

Comment:
Can you change the option to be `fade`. It may just be me, but it's a little harder to think about what value to put here, if you have the negative version as the option.

Also, can you put the default in brackets, like this? `[options.fade=true]` (And remove it from the description text.) That way, the generated docs display the default correctly.